.ai/auto-review/github_reviewer.py

```diff
@@ -112,6 +133,22 @@ def post_line_comment(github: GitHub, file: str, text:str, line: int):
     except RepositoryError as e:
         Log.print_red("Failed line comment", e)
         return False
+    
+def post_line_comment_with_diff_hunks(github: GitHub, file: str, text:str, line: int, diff_hunk: str):
+    Log.print_green("Posting line", file, line, text)
+    try:
+        git_response = github.post_comment_to_line_with_diff_hunks(
+            text=text, 
```

Comment:
141 : Incorrect argument: `post_line_comment_with_diff_hunks` is called with diff_hunk which is list of diff hunks, but should be a diff hunk string. : Pass each hunk in `diff_hunks` to the function separately in a loop or change the function to accept a list of diff hunks.